tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/CircuitBreakerMetricTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.metrics;
+
+import static org.eclipse.microprofile.fault.tolerance.tck.metrics.Exceptions.expectCbOpen;
+import static org.eclipse.microprofile.fault.tolerance.tck.metrics.Exceptions.expectTestException;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricBean.Result;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.testng.annotations.BeforeTest;
+
+public class CircuitBreakerMetricTest extends Arquillian {
+
+    public static WebArchive deploy() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "ftMetricCircuitBreaker.war")
+                .addClasses(CircuitBreakerMetricBean.class, MetricGetter.class);
+        return war;
+    }
+    
+    @Inject private CircuitBreakerMetricBean cbBean;
+    
+    @BeforeTest
+    public void closeTheCircuit() throws Exception {
+        // Assume the circuit is open
+        // Wait for it to half close, then close it with passing work
+        Thread.sleep(1500);
```

Comment:
Could be improved to sleep only if the circuit is really open. The state can be retrieved from `MetricGetter`.

Reply:
That would have been true, although we just removed that metric in the meeting this afternoon because it would be very easy to create a misleading graph unless it was sampled very rapidly.

We could possibly check the accumulated time metrics instead, as long as we were careful not to hang the test if those metrics were broken in the implementation under test.